tomviz/modules/ModuleSlice.h

```diff
@@ -44,6 +45,7 @@ class ModuleSlice : public Module
   QJsonObject serialize() const override;
   bool deserialize(const QJsonObject& json) override;
   bool isColorMapNeeded() const override { return true; }
+  bool isColorMapOpaque() const override { return m_opaqueMap; }
```

Comment:
I think this naming is odd, maybe more isOpacityMapped would get across what we mean, when enabled it could be fully opaque, or fully transparent, whatever the color map asks for,

Reply:
m_mapOpacity would probably be simpler/convey the meaning more clearly.